packages/cli-plugin-scaffold-app-graphql/index.js

```diff
@@ -0,0 +1,249 @@
+					throw new Error(`Destination folder ${fullPackageLocation} already exists!`);
+				}
+				
+				await fs.mkdirSync(fullPackageLocation, {recursive: true});
+				
+				// Get base TS config path
+				const baseTsConfigPath = path
+					.relative(
+						fullPackageLocation,
+						findUp.sync("tsconfig.json", {
+							cwd: fullPackageLocation
+						})
+					)
+					.replace(/\\/g, "/");
+				
+				// Copy template files
+				await ncp(sourceFolder, fullPackageLocation);
+				
+				// Replace generic "Entity" with received "input.existingDataModelName" or "input.dataModelName" argument.
+				const entity = {
+					plural: pluralize(Case.camel(modelName)),
+					singular: pluralize.singular(Case.camel(modelName))
+				};
+				
+				const codeReplacements = [
+					{find: "entities", replaceWith: Case.camel(entity.plural)},
+					{find: "Entities", replaceWith: Case.pascal(entity.plural)},
+					{find: "ENTITIES", replaceWith: Case.constant(entity.plural)},
+					{find: "entity", replaceWith: Case.camel(entity.singular)},
+					{find: "Entity", replaceWith: Case.pascal(entity.singular)},
+					{find: "ENTITY", replaceWith: Case.constant(entity.singular)}
+				];
+				
+				replaceInPath(path.join(fullPackageLocation, "**/*.ts"), codeReplacements);
+				replaceInPath(path.join(fullPackageLocation, "**/*.tsx"), codeReplacements);
+				
+				// Make sure to also rename base file names.
+				const fileNameReplacements = [
+					{
+						find: "views/EntitiesDataList.tsx",
+						replaceWith: `views/${Case.pascal(entity.plural)}DataList.tsx`,
+					},
+					{
+						find: "views/Entities.tsx",
+						replaceWith: `views/${Case.pascal(entity.plural)}.tsx`,
+					},
+					{
+						find: "views/EntityForm.tsx",
+						replaceWith: `views/${Case.pascal(entity.singular)}Form.tsx`,
+					},
+					{
+						find: "example.tsconfig.json",
+						replaceWith: "tsconfig.json"
+					}
+				];
+				
+				for (const key in fileNameReplacements) {
+					if(!fileNameReplacements.hasOwnProperty(key)) {
+						continue;
+					}
+					const fileNameReplacement = fileNameReplacements[key];
+					fs.renameSync(
+						path.join(fullPackageLocation, fileNameReplacement.find),
+						path.join(fullPackageLocation, fileNameReplacement.replaceWith)
+					);
+				}
+				
+				// Update the package's name
+				const packageJsonPath = path.resolve(fullPackageLocation, "package.json");
+				let packageJson = fs.readFileSync(packageJsonPath, "utf8");
+				packageJson = packageJson.replace(/\[PACKAGE_NAME\]/g, packageName);
+				fs.writeFileSync(packageJsonPath, packageJson);
+				
+				// Update tsconfig "extends" path
+				const tsConfigPath = path.join(fullPackageLocation, "tsconfig.json");
+				const tsconfig = require(tsConfigPath);
+				tsconfig.extends = baseTsConfigPath;
+				fs.writeFileSync(tsConfigPath, JSON.stringify(tsconfig, null, 2));
+				
+				oraSpinner.info('You must register generated GraphQL app in your app factory "plugins" property');
```

Comment:
Need to clarify this message a bit more. Maybe even print a path to the actual app, like `apps/admin/src/App.tsx`, so the user knows where exactly to go next.

Let's tweak the message: 
```
You must register the generated plugins in order for them to appear in your app. 
Open your ${green(PATH_TO_APP_TSX)} and pass your new plugins to the app template via the `plugins` option.

Learn more about Webiny app development at https://docs.webiny.com/docs/app-development/introduction.
```